app/models/provider.rb

```diff
@@ -58,6 +58,12 @@ def save!
     refresh!
   end
 
+  def self.custom_provider_id(config)
```

Comment:
this might not be necessary after a rebase since its already been implemented for creation of custom packages?

Reply:
Ok, I had a similar method - updated to use package version method (from rebase) but slightly different rm api request so I needed to re-record tests